From 5f09787e92fc2d86b71dd716a6d335c15850532c Mon Sep 17 00:00:00 2001
From: shengfei Xu <xsf@rock-chips.com>
Date: Sun, 30 Sep 2018 09:07:44 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: add PSCI node

Add PSCI node for RK1808 SoC, and cpu node enable-method property is
set to "psci".

Change-Id: I08c0f49633ec65c6b054c683556add168dd89e48
Signed-off-by: shengfei Xu <xsf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 0a81fa67fd15..c43094230564 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -42,6 +42,7 @@
 			device_type = "cpu";
 			compatible = "arm,cortex-a35", "arm,armv8";
 			reg = <0x0 0x0>;
+			enable-method = "psci";
 			clocks = <&cru ARMCLK>;
 		};
 
@@ -49,6 +50,7 @@
 			device_type = "cpu";
 			compatible = "arm,cortex-a35", "arm,armv8";
 			reg = <0x0 0x1>;
+			enable-method = "psci";
 			clocks = <&cru ARMCLK>;
 		};
 	};
@@ -67,6 +69,11 @@
 		#clock-cells = <0>;
 	};
 
+	psci {
+		compatible = "arm,psci-1.0";
+		method = "smc";
+	};
+
 	timer {
 		compatible = "arm,armv8-timer";
 		interrupts = <GIC_PPI 13 IRQ_TYPE_LEVEL_LOW>,
-- 
2.35.3

